### PR TITLE
[CodingStyle] Add NullifyUnionNullableRector

### PR DESCRIPTION
**Pull request by samsonasik**
To convert already typed `null|stdClass` to `?stdClass`

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

**Comment by samsonasik** [comment]
I make it optional by unregister from coding-style config set.

**Comment by samsonasik** [comment]
I register NullifyUnionNullableRector to MissingInSetCommand https://github.com/rectorphp/rector-src/pull/3231/commits/e0fa88c528b38317896233a86cc3b56b381577f8

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

File: packages/BetterPhpDocParser/PhpDocParser/ConstExprClassNameDecorator.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function decorate(PhpDocNode $phpDocNode, PhpNode $phpNode): void
     {
         $this->phpDocNodeTraverser->traverseWithCallable($phpDocNode, '', function (Node $node) use (
             $phpNode
-        ): Node|null {
+        ): ?Node {
```
Comment (TomasVotruba) [inline packages/BetterPhpDocParser/PhpDocParser/ConstExprClassNameDecorator.php]: The rule changes should be reverted as well

Comment (samsonasik) [reply packages/BetterPhpDocParser/PhpDocParser/ConstExprClassNameDecorator.php]: Ok, reverted https://github.com/rectorphp/rector-src/pull/3231/commits/755100ccf04af7b7bd8d47f762e840603d37cd1d https://github.com/rectorphp/rector-src/pull/3231/commits/c74239f480cd1f0cf3f9fc92c4708fd9e9d09f1f

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**